### ISSUE TITLE
scheduler.step() 问题

### ISSUE THREAD

**Issue by 1611064578** [boy56/PLMEE#19]
您好，我试了一下。跑起来了，还挺快的，但是显示`scheduler.step()` 问题。
/home/hello/ .local/1ib/python3 . 8/site-packages/ torch/ optim/1r scheduLer . py:156: UserWarning: The epoch parameter in' scheduLer . step()was not necessary and is being deprecated where possible. PLease use、 scheduLer . step() to step the scheduLer. During the deprecation, if epoch is different from None, the closed form is used instead 0the new chainable form, where available. PLease open an issue if you are unabLe to replicate your use case: https://github .com/ pytorch/ pytorch/issues/new/ choose . warnings . warn(EPOCH_DEPRECATION _WARNING，UserWarning)
翻译过来就是scheduler.step（） 中的 epoch 参数不是必需的，在可能的情况下被弃用。请使用 scheduler.step（） 来步进调度程序。
这是需要另外修改的吗？后天开组会了，希望您帮忙解释一下，谢谢您

**Comment by boy56** [comment]
> 您好，我试了一下。跑起来了，还挺快的，但是显示`scheduler.step()` 问题。 /home/hello/ .local/1ib/python3 . 8/site-packages/ torch/ optim/1r scheduLer . py:156: UserWarning: The epoch parameter in' scheduLer . step()was not necessary and is being deprecated where possible. PLease use、 scheduLer . step() to step the scheduLer. During the deprecation, if epoch is different from None, the closed form is used instead 0the new chainable form, where available. PLease open an issue if you are unabLe to replicate your use case: https://github .com/ pytorch/ pytorch/issues/new/ choose . warnings . warn(EPOCH_DEPRECATION _WARNING，UserWarning) 翻译过来就是scheduler.step（） 中的 epoch 参数不是必需的，在可能的情况下被弃用。请使用 scheduler.step（） 来步进调度程序。 这是需要另外修改的吗？后天开组会了，希望您帮忙解释一下，谢谢您

这种问题我也没遇到过，但我怀疑是torch版本问题。以及，一般warning不用管，能跑就行。

**Comment by 1611064578** [comment]
他一次是跑两个吗？我这里显示出现warning后自己又跑了一遍。如果是跑两个的话，他跑完我除了能看到这个正确率 还能得到什么呢？谢谢您






------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2023年2月22日(星期三) 下午3:30
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [boy56/PLMEE] scheduler.step() 问题 (Issue #19)





  
您好，我试了一下。跑起来了，还挺快的，但是显示scheduler.step() 问题。 /home/hello/ .local/1ib/python3 . 8/site-packages/ torch/ optim/1r scheduLer . py:156: UserWarning: The epoch parameter in' scheduLer . step()was not necessary and is being deprecated where possible. PLease use、 scheduLer . step() to step the scheduLer. During the deprecation, if epoch is different from None, the closed form is used instead 0the new chainable form, where available. PLease open an issue if you are unabLe to replicate your use case: https://github .com/ pytorch/ pytorch/issues/new/ choose . warnings . warn(EPOCH_DEPRECATION _WARNING，UserWarning) 翻译过来就是scheduler.step（） 中的 epoch 参数不是必需的，在可能的情况下被弃用。请使用 scheduler.step（） 来步进调度程序。 这是需要另外修改的吗？后天开组会了，希望您帮忙解释一下，谢谢您↳
  
这种问题我也没遇到过，但我怀疑是torch版本问题。以及，一般warning不用管，能跑就行。

**Comment by boy56** [comment]
分训练和推理单独跑就行，得到的肯定是训练好的模型啊，然后做推理一类的。建议读一下main函数，一般功能都在main函数里。
